src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -206,6 +211,13 @@ def source_waveform_type(request):
 
 
 def test_source_waveform_parallel_site_unique(multi_instrument_session, source_waveform_type):
+    '''Test methods for generating and capturing unique waveforms to multiple sites in parallel.
```

Comment:
Test methods for using source waveform with parallel sourcing and site-unique data mapping.

Reply:
Updated.